workitem/workitemtype.go

```diff
@@ -26,6 +26,7 @@ const (
 	SystemCreatedAt           = "system.created_at"
 	SystemIteration           = "system.iteration"
 	SystemArea                = "system.area"
+	SystemOrder               = "order"
```

Comment:
I think this is for no particular reason. @DhritiShikhar please, rename as @aslakknutsen suggested.